modules/chapter3/pages/section1.adoc

```diff
@@ -1,3 +1,68 @@
+
+image::elyra_jupyter.png[Jupyter with elyra]
+
+Elyra supports two types of components: generic components and custom components. A pipeline that utilizes only generic components is called a generic pipeline, whereas a pipeline that utilizes generic components and/or custom components is referred to as runtime-specific pipeline.
+
+The Elyra Visual Pipeline editor lets you assemble pipelines by dragging and dropping supported files onto the canvas and defining their dependencies. After you've assembled the pipeline and are ready to run it, the editor takes care of generating the Apache Airflow DAG code on the fly, eliminating the need for any coding.
+
+== Creating a Data Science Pipeline with Elyra
+
+Pipelines are created in Elyra with the Visual Pipeline Editor by:
+
+* Launch JupyterLab with the Elyra extension installed.
+* Create a new pipeline by clicking on the Elyra pipeline editor icon.
+* Add each node to the pipeline by dragging and dropping notebooks or scripts from the file browser onto the pipeline editor canvas.
+* Connect the nodes to define the flow of execution.
+* Configure each node by double-clicking on it and setting the appropriate runtime image and file dependencies.
+* Once the pipeline is complete, you can run it locally or submit it to run on Kubeflow Pipelines or Apache Airflow.
+
+== Creating a runtime configuration
+
+A runtime configuration provides Elyra access to external resources, such as Kubeflow Pipelines or Apache Airflow for scalable pipeline execution.You can manage runtime configurations using the JupyterLab UI or the Elyra CLI.
+
+To create a runtime configuration:
+
+* Open the Runtimes panel.
+* Click + to add a new runtime configuration and choose the desired runtime configuration type, e.g. Kubeflow Pipelines or Apache Airflow.
+* Provide a runtime configuration display name, an optional description, and tag the configuration to make it more easily discoverable.
+image::Create_runtime_configuration.png
+* Enter the Kubeflow Pipelines or Apache Airflow deployment information. Refer to section Kubeflow Pipelines configuration settings (https://elyra.readthedocs.io/en/latest/user_guide/runtime-conf.html#kubeflow-pipelines-configuration-settings) or Apache Airflow configuration settings(https://elyra.readthedocs.io/en/latest/user_guide/runtime-conf.html#apache-airflow-configuration-settings) for details.
+* Enter the Cloud Storage connectivity information. Refer to section Cloud Storage settings for details.
+* Save the runtime configuration. The new entry is displayed in the list.
+* Expand the entry and verify that you can access the Kubeflow Pipelines or Apache Airflow GUI and the Cloud Storage GUI using the displayed links.
+
+== Pipeline to Predict weather
```

Comment:
As discussed, let's use this Elyra pipeline example that is also covered in the KFP section:
https://github.com/mamurak/os-mlops/tree/main/notebooks/fraud-detection-onnx/offline-scoring

The idea could be to provide the basic Python modules and go through creating the Elyra pipeline in the lab.